doc/admin-api-cn.md

```diff
@@ -19,11 +19,13 @@
 
 # 目录
 
-* [Route](#route)
-* [Service](#service)
-* [Consumer](#consumer)
-* [Upstream](#upstream)
-* [SSL](#ssl)
+- [目录](#%e7%9b%ae%e5%bd%95)
```

Comment:
we'd better use Chinese here. `#%e7%9b%ae%e5%bd%95` is not a good choice.

Reply:
Hi, after submitting last night I noticed that due to the vs code markdown plugin, it automatically generates a list of headings that overwrite the past ones, I need to fix that part!